### ISSUE TITLE
Has anyone successfully trained SiamMask?

### ISSUE THREAD

**Issue by xieyunjiao** [foolwood/SiamMask#112]
There are many and large training sets. Is there any way to adjust the training data set?
After the adjustment,has anyone successfully trained?

**Comment by Damon2019** [comment]
The dataset provided is too large.I want to train my own voc or coco dataset.Have you succeeded?thanks!

**Comment by xieyunjiao** [comment]
> The dataset provided is too large.I want to train my own voc or coco dataset.Have you succeeded?thanks!

I tried to resize the data set, but it went wrong and I haven't trained it yet.

**Comment by Damon2019** [comment]
> > The dataset provided is too large.I want to train my own voc or coco dataset.Have you succeeded?thanks!
> 
> I tried to resize the data set, but it went wrong and I haven't trained it yet.

I am still groping.  Let us together strive to work together!

**Comment by Damon2019** [comment]
> > The dataset provided is too large.I want to train my own voc or coco dataset.Have you succeeded?thanks!
> 
> I tried to resize the data set, but it went wrong and I haven't trained it yet.

Hello, have you succeeded yet? Can I add your contact information?

**Comment by xieyunjiao** [comment]
> > > The dataset provided is too large.I want to train my own voc or coco dataset.Have you succeeded?thanks!
> > 
> > 
> > I tried to resize the data set, but it went wrong and I haven't trained it yet.
> 
> Hello, have you succeeded yet? Can I add your contact information?
xiexietruly@gmail.com

**Comment by vamsi-pujari** [comment]
@xieyunjiao @Damon2019 @foolwood Hey I want to train on my own dataset. In Siammask/data/coco/annotations_trainval2017/instances_val2017.json 
1.Whats the format of segmentation for each image at 'Annotations>segmentation'
2.Is there any application where i can label my data for segmentation like Labelimg for boundingbox.
